data/json/items/classes/gun.json

```diff
@@ -33,25 +33,61 @@
     "type": "GUN",
     "name": "base pistol",
     "skill": "pistol",
-    "valid_mod_locations": [ [ "accessories", 2 ], [ "barrel", 1 ], [ "bore", 1 ], [ "grip", 1 ], [ "mechanism", 4 ], [ "magazine", 1 ], [ "muzzle", 1 ], [ "rail", 1 ], [ "sights", 1 ], [ "stock", 1 ], [ "underbarrel", 1 ] ]
+    "valid_mod_locations": [
+      [ "accessories", 2 ],
+      [ "barrel", 1 ],
+      [ "bore", 1 ],
+      [ "grip", 1 ],
+      [ "mechanism", 4 ],
+      [ "magazine", 1 ],
+      [ "muzzle", 1 ],
+      [ "rail", 1 ],
+      [ "sights", 1 ],
+      [ "stock", 1 ],
+      [ "underbarrel", 1 ]
+    ]
```

Comment:
This does look bad. If it was just the base abstract guns it would be tolerable, but there are tons of guns not using this system.

Reply:
There is no way that wrapping the line off the edge of the screen (225 characters!) is more readable than writing one element per line.